src/renderer/dx/DxRenderer.cpp

```diff
@@ -1146,8 +1051,8 @@ void DxEngine::_InvalidOr(RECT rc) noexcept
         {
             HRESULT hr = S_OK;
 
-            hr = _dxgiSwapChain->Present(1, 0);
-            /*hr = _dxgiSwapChain->Present1(1, 0, &_presentParams);*/
+            /*hr = _dxgiSwapChain->Present(1, 0);*/
+            hr = _dxgiSwapChain->Present1(1, 0, &_presentParams);

```

Comment:
We normally don't leave behind dead code, but I'm not comfortable taking out the `Present` call yet as we might hit something that we aren't expecting.